[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/technical-guides/test-results-biztalk-server-key-performance-indicators.md

This topic summarizes BizTalk Server Key Performance Indicators (KPI) observed during the test scenarios. Specifically these tests evaluated throughput as measured by the "**BizTalk:Messaging/Documents processed/Sec**" performance monitor counter, and latency, as measured by the Visual studio client response time.

## Summary of BizTalk Server Key Performance Indicators

>  For each scenario the physical machines were restricted so that number of logical processors and virtual processors was equivalent. This was done using the /maxmem and /numproc boot.ini switches. For more information about using these switches, see [Boot INI Options Reference](https://go.microsoft.com/fwlink/?LinkId=122139.).

Reviewer: ```suggestion
For each scenario the physical machines were restricted so that number of logical processors and virtual processors was equivalent. This was done using the **/maxmem** and **/numproc** switches for the **boot.ini** file. For more information, see [Boot Options in a Boot.ini File](/windows-hardware/drivers/devtest/boot-options-in-a-boot-ini-file).
```

>  **Comparison of BizTalk Server Key Performance Indicators –** Running BizTalk Server on a Hyper-V virtual machine provided approximately 95% of the throughput and latency performance of BizTalk Server on physical hardware for this test scenario. Because of the stateless nature of [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)], additional BizTalk Server virtual machines can be easily added to the environment as required to provide scale out and increase the overall performance of the system. Creating and adding additional BizTalk Server to the environment can be accomplished by using the sysprep utility to generate new images from a base image.

Reviewer: ```suggestion
### Comparison of BizTalk Server Key Performance Indicators

For this test scenario, running BizTalk Server on a Hyper-V virtual machine provided approximately 95% of the throughput and latency performance of BizTalk Server on physical hardware. To provide scale out and increase the overall performance of the system, you can easily add more BizTalk Server virtual machines to the environment as required, due to the stateless nature of [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)]. You can create and add more BizTalk Server virtual machines to the environment by using the **SysPrep** utility to generate new images from a base image.
```

> [!NOTE]

> >  A sysprep answer file and scripts are provided with BizTalk Server to accommodate using sysprep to create additional images from an existing image of a computer that has BizTalk Server installed. These sample scripts are designed for use with BizTalk Server installed on 32-bit and 64-bit versions of Windows Server 2008 only. For more information see the BizTalk Server online documentation.

Reviewer: ```suggestion
> You can create more images based on an existing image of a computer that has BizTalk Server installed. 
> You can run SysPrep against a base image by using a SysPrep answer file and scripts provided with BizTalk Server. 
> These sample scripts are designed for use only with BizTalk Server installed on 32-bit and 64-bit versions of 
> Windows Server 2008. For more information see [Sysprep a BizTalk Server VHD (BizTalk Server Sample)](/biztalk/core/sysprep-a-biztalk-server-vhd-biztalk-server-sample).
```

 Provisioning, consolidation, and management of virtual machines can be significantly expedited through the use of System Center Virtual Machine Manager (VMM). For more information, see [System Center Virtual Machine Manager](/system-center/vmm/whats-new-in-vmm).

>  The results obtained in this performance lab show a marked improvement from the performance achieved when running [!INCLUDE[btsbiztalkserver2006r2](../includes/btsbiztalkserver2006r2-md.md)] on [!INCLUDE[btsWinSvr2k3](../includes/btswinsvr2k3-md.md)] in a Hyper-V virtual machine. Running [!INCLUDE[btsbiztalkserver2006r2](../includes/btsbiztalkserver2006r2-md.md)] on a Hyper-V virtual machine provided approximately 75% of the throughput and latency performance of [!INCLUDE[btsbiztalkserver2006r2](../includes/btsbiztalkserver2006r2-md.md)] on physical hardware versus the approximately 95% performance observed when running BizTalk Server and [!INCLUDE[btsWinSvr2k8](../includes/btswinsvr2k8-md.md)] on Hyper-V virtual machines. This improved performance is largely attributable to the improved performance of [!INCLUDE[btsWinSvr2k8](../includes/btswinsvr2k8-md.md)] when running as a guest operating system on Hyper-V. The related performance comparison from the [!INCLUDE[btsbiztalkserver2006r2](../includes/btsbiztalkserver2006r2-md.md)] [Hyper-V guide](https://go.microsoft.com/fwlink/?LinkId=147144) is available.

Reviewer: ```suggestion
The results obtained in this performance lab show a marked improvement from the performance achieved when you run [!INCLUDE[btsbiztalkserver2006r2](../includes/btsbiztalkserver2006r2-md.md)] on [!INCLUDE[btsWinSvr2k3](../includes/btswinsvr2k3-md.md)] in a Hyper-V virtual machine. Running [!INCLUDE[btsbiztalkserver2006r2](../includes/btsbiztalkserver2006r2-md.md)] on a Hyper-V virtual machine provided approximately 75% of the throughput and latency performance of [!INCLUDE[btsbiztalkserver2006r2](../includes/btsbiztalkserver2006r2-md.md)] on physical hardware versus the approximately 95% performance observed when running BizTalk Server and [!INCLUDE[btsWinSvr2k8](../includes/btswinsvr2k8-md.md)] on Hyper-V virtual machines. This improved performance is largely attributable to the improved performance of [!INCLUDE[btsWinSvr2k8](../includes/btswinsvr2k8-md.md)] when running as a guest operating system on Hyper-V. The related performance comparison is available from the [!INCLUDE[btsbiztalkserver2006r2](../includes/btsbiztalkserver2006r2-md.md)] [Hyper-V guide](/biztalk/technical-guides/biztalk-server-2010-hyper-v-guide).
```

|\BizTalk:Messaging\Documents processed/Sec|94.3%|79.8%|67%|
|Latency as measured by the Visual Studio client|94.3%|79.7%|66.9%|

>  For more information about how to optimize the performance of a BizTalk Server solution, see [BizTalk Server Performance Optimizations Guide](https://go.microsoft.com/fwlink/?LinkId=122477).

Reviewer: ```suggestion
 For more information about how to optimize the performance of a BizTalk Server solution, see [Performance and Capacity Planning](/biztalk/core/performance-and-capacity-planning).
```
